src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator {
+    private String colorFigure;
+
+    public Figure(String colorFigure) {
+        this.colorFigure = colorFigure;
```

Comment:
We are **inside** Figure - no need to repeat this word
```suggestion
    private String color;

    public Figure(String color) {
        this.color = color;
```